test/e2e/k8s_util.go

```diff
@@ -332,6 +334,68 @@ func decidePingProbeResult(stdout string, probeNum int) PodConnectivityMark {
 	}
 	return Error
 }
+func (k *KubernetesUtils) digDnSCustom(
```

Comment:
This function can probably be greatly simplified by using the `+short` dig option:
```
C02F16ASMD6R:antrea abas$ dig @8.8.8.8 www.google.com

; <<>> DiG 9.10.6 <<>> @8.8.8.8 www.google.com
; (1 server found)
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 64670
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;www.google.com.			IN	A

;; ANSWER SECTION:
www.google.com.		298	IN	A	142.251.46.196

;; Query time: 110 msec
;; SERVER: 8.8.8.8#53(8.8.8.8)
;; WHEN: Thu Sep 26 16:00:18 PDT 2024
;; MSG SIZE  rcvd: 59

C02F16ASMD6R:antrea abas$ dig @8.8.8.8 +short www.google.com
142.251.46.196
```